### DISCUSSION TITLE
Dynamic Recipients options and Admin Notifications

### DISCUSSION THREAD

**Discussion by jrrdnx** [solspace/craft-freeform#294] [Feature Requests]
Currently the documentation doesn't indicate that if a Dynamic Recipients field is being used, email notifications will still be sent to any addresses entered into the Admin Notifications field. I'm not sure how I came to be under that impression, but that's my fault.

https://docs.solspace.com/craft/freeform/v2/overview/email-notifications.html#types-of-email-notifications
https://docs.solspace.com/craft/freeform/v2/overview/forms-composer.html#admin-email-notifications
https://docs.solspace.com/craft/freeform/v3/overview/fields.html#dynamic-recipients

Related, could an option be added to the Dynamic Recipients field to toggle this behavior? I could see instances where a client wants only Dynamic Recipients notified and others where they always want an Admin notified in addition to, but the Admin is always notified if the DR field is optional and no selection is made.

**Comment by kjmartens** [comment]
Hi @jrrdnx,

What if you just included an option in the Dynamic Recipients field that includes the email addresses you have in the Admin Notify area (and have nothing in the Admin Notify area), and set that as the default option for the Dynamic Recipients field? It seems it would handle this scenario. 🙂

**Comment by kjmartens** [comment]
@jrrdnx I just wanted to let you know that this will be available in the next major version of Freeform. 🙂

**Comment by kjmartens** [comment]
Email notifications have been overhauled in Freeform 5 for Craft 4 (now in beta), and Conditional Notifications are now available. 🙂
